homeassistant/helpers/translation.py

```diff
@@ -21,7 +21,7 @@
 TRANSLATION_FLATTEN_CACHE = "translation_flatten_cache"
 
 MOVED_TRANSLATIONS_DIRECTORY_MSG = (
-    "%s: the '.translations' directory has been moved, the new name is 'translations', "
+    "%s: the 'translations' directory has been moved, the new name is 'translations', "
```

Comment:
Please don't change this file.

Reply:
Ouch, sorry ! I trusted the refactoring of pyCharm too much, sorry for that !
I reverted the unwanted changes.